packages/flutter/lib/src/rendering/custom_layout.dart

```diff
@@ -41,26 +41,42 @@ class MultiChildLayoutParentData extends ContainerBoxParentData<RenderBox> {
 /// Used with [CustomMultiChildLayout], the widget for the
 /// [RenderCustomMultiChildLayoutBox] render object.
 ///
-/// ## Example
+/// Each child must be wrapped in a [LayoutId] widget to assign the id that
+/// identifies it to the delegate.
+///
+/// ## Example code
 ///
 /// Below is an example implementation of [performLayout] that causes one widget
-/// to be the same size as another:
+/// (the follower) to be the same size as another (the leader):
 ///
 /// ```dart
-/// @override
-/// void performLayout(Size size) {
-///   Size followerSize = Size.zero;
-///
-///   if (hasChild(_Slots.leader) {
-///     followerSize = layoutChild(_Slots.leader, new BoxConstraints.loose(size));
-///     positionChild(_Slots.leader, Offset.zero);
-///   }
-///
-///   if (hasChild(_Slots.follower)) {
-///     layoutChild(_Slots.follower, new BoxConstraints.tight(followerSize));
-///     positionChild(_Slots.follower, new Offset(size.width - followerSize.width,
-///                                               size.height - followerSize.height));
+/// // Define your own slot numbers, depending upon the id assigned by LayoutId.
+/// // Typical usage is to define an enum like the one below, and use those
+/// // values as the ids.
+/// enum _Slot {
+///   leader,
+///   follower,
+/// }
+/// 
+/// class FollowTheLeader extends MultiChildLayoutDelegate {
+///   @override
+///   void performLayout(Size size) {
+///     Size followerSize = Size.zero;
+/// 
+///     if (hasChild(_Slot.leader)) {
+///       followerSize = layoutChild(_Slot.leader, new BoxConstraints.loose(size));
```

Comment:
Would it make more sense for this var to be named `leaderSize`?
- `layoutChild` returns the size of the widget it processes.
- `follower` is constrained/positioned according to the `leader`'s size.

Reply:
i agree that leaderSize makes more sense

Reply:
OK, done.